### PR TITLE
Use Composer path repositories to install packages in parent project.

### PR DESCRIPTION
**Pull request by mortenscheel**
Hi, thanks for an excellent package.

I've made a few changes that I think makes the workflow a bit simpler and more intuitive.
By using path repositories in the parent project, the packages can be installed exactly like any other package.
Here are the benefits as I see them:
- No need to install/update dependencies in the package sub-folders. Everything is managed from the project root.
- Service providers are discovered automatically during installation.
- Classes in the package namespaces are autoloaded automatically just like any other package.
- Having only one `vendor/` folder means no duplicate dependencies, which tend to confuse IDEs like PhpStorm.

Let me know if anything needs to be changed.

### CODE REVIEW

File: src/Commands/DisablePackage.php [hunk 1]
```diff
@@ -72,11 +72,9 @@ public function handle()
         $this->info('Disabling package '.$this->conveyor->vendor().'\\'.$this->conveyor->package().'...');
         $this->makeProgress();
 
-        // Composer dump-autoload to remove service provider
-        $this->info('Dumping autoloads and undiscovering package...');
-        $this->wrapping->removeFromComposer($this->conveyor->vendor(), $this->conveyor->package());
-        $this->wrapping->removeFromProviders($this->conveyor->vendor(), $this->conveyor->package());
-        $this->conveyor->dumpAutoloads();
```
Comment (Jeroen-G) [inline src/Commands/DisablePackage.php]: Are you sure the autoloads are updated without this?

**Comment by Jeroen-G** [comment]
You are amazing! I wanted this for a while now but didn't find the time for it. I just merged #74 in master. Could you merge master in your branch to see how the couple of tests turn out?

File: src/Commands/DisablePackage.php (hunk 1 above)
Comment (mortenscheel) [reply src/Commands/DisablePackage.php]: Yes they are updated automatically each time `composer require` or `composer remove` are run.

**Comment by mortenscheel** [comment]
It seems that Orchestra/Testbench didn't like my changes at all, so I've had to modify the testing environment a bit.

Honestly I'm not super experienced in the testing department, so feel free to educate me if I've done something stupid.

The main issue was that I'm now actually installing dependencies in the Testbench folder (something I'm pretty sure it was never designed for), and resetting the test environment between each test was a hassle.

In the end I decided to just wipe the entire testbench and recreate it between tests. This would obviously be insanely slow if Composer had to pull in the entire Laravel framework each time, so in stead I've made a "test app template", in which `illuminate/support` is already installed. The template is only created once, and is relatively fast to copy.

All the tests are green on my machine, but Travis shows some errors, which I assume are because PhpUnit 8 is required now.
Is there any way to fix the StyleCI errors automatically?

**Comment by Jeroen-G** [comment]
I really appreciate your work! I'll look into the code and the tests asap. When the PR is merged styleCI will place it's own PR to fix the styling, so no worries there.

**Comment by mortenscheel** [comment]
You're welcome. I've been wanting to contribute to something for a while, and I had a few days off work, with nothing better to do :)

I've added another feature, but maybe I should wait and send it in a separate pull-request later?
The `packager:git` command creates a vcs repository in composer.json, installs the package to `vendor`, and finally symlinks the package to `packages/{vendor}/{package}`.
Pulling in changes from upstream can be done with a simple `composer update vendor/package`.
I also added a 'Git status' column to the output of `packager:list`, showing how many commits you're ahead or behind.

**Comment by Jeroen-G** [comment]
Yeah, let's save that for the next PR ;)

**Comment by Jeroen-G** [comment]
Tests are green! 🚀

**Comment by Jeroen-G** [comment]
Thank you for your work, it is now in master! I think this should be in a major release since it more or less changes the core thing it does. That means that at this point I'm open to your other PR and any other wild idea you got 😄

**Comment by mortenscheel** [comment]
Thanks man 👍 
I've submitted the other feature as a new PR.

**Pull request merged**